sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/csv/CSVSuite.scala

```diff
@@ -805,6 +805,22 @@ abstract class CSVSuite
     }
   }
 
+  test("SPARK-37575: null values should not reflect to any characters by default") {
```

Comment:
Could you make test's title more clear like PR's title, please. For now, it might confuses other devs. Does null reflect to any character in write, right now?

Reply:
Fine, I made it more clear.